### DISCUSSION TITLE
[Feature request] Web clipper extension for browser

### DISCUSSION THREAD

**Discussion by xBlady** [docmost/docmost#1636] [Ideas]
Hello!

Amazing app!

It would be great to have web clipper extension for chrome, firefox etc. that would be able to capture the page i'm on, fragment of the text and fetch website title and link into it. There might be no need to create separate extension, as there are many open source web clippers available, the only thing needed would be easy configuration of it and quick documentation article how to use it.

**Comment by solodu** [comment]
+1
